dev-client/__tests__/integration/EditSiteNoteScreen.test.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {testState} from '@testing/integration/data';
+import {render} from '@testing/integration/utils';
+
+import {EditSiteNoteScreen} from 'terraso-mobile-client/screens/SiteNotesScreen/EditSiteNoteScreen';
+
+const mockedNavigate = jest.fn();
+jest.mock('terraso-mobile-client/navigation/hooks/useNavigation', () => {
+  const actualNav = jest.requireActual(
+    'terraso-mobile-client/navigation/hooks/useNavigation',
+  );
+  return {
+    ...actualNav,
+    useNavigation: () => ({navigate: mockedNavigate}),
+  };
+});
+
+afterEach(() => {
```

Comment:
I think this also needs to go inside the describe().

Reply:
Since there's only one describe block in the test file, it shouldn't make a difference; it'll be called either way. Source: https://jestjs.io/docs/setup-teardown 

Since the `mockedNavigate` was declared outside the describe block (and didn't seem to work inside it for reasons I did not investigate) I put it in the same scope, but I really don't have strong feelings about it. Lmk if you still have a preference.